docs/application/dotnet/guides/nui/customview.md

```diff
@@ -82,9 +82,144 @@ To create a custom view:
     }
     ```
 
-    The following table lists other important custom view methods that you can use to manage the view.
+5. Define the constructor to create an instance of the custom view. In this case, the `ContactView` is created with the `contactName` parameter, and the resource directory path points to the icons. Additionally, the `base` constructor has to be called with the type name and view behavior as described in the subsequent section. In a nonstatic constructor, `Visuals` used to create a custom view should be created. For instance, in the following `ContactView` example one label, background, and a few icons are shown:
 
-    **Table: Custom view methods**
+    ![Contact View](media/contact.png)
+
+    ```
+    public ContactView(string contactName, string resDir) : base(typeof(ContactView).Name, CustomViewBehaviour.ViewBehaviourDefault)
+        {
+            CreateBackground(new Vector4(1.0f, 1.0f, 1.0f, 1.0f));
+            CreateIcon(resDir + "/images/cbg.png", 10.0f, 5.0f, 100.0f, 100.0f, CONTACT_BG_ICON_INDEX);
+            CreateIcon(resDir + "/images/contact.png", 10.0f, 5.0f, 100.0f, 100.0f, CONTACT_ICON_INDEX);
+            CreateIcon(resDir + "/images/edit.png", 130.0f, 40.0f, 50.0f, 50.0f, CONTACT_EDIT_INDEX);
+            CreateIcon(resDir + "/images/favorite.png", 180.0f, 40.0f, 50.0f, 50.0f, CONTACT_FAVORITE_INDEX);
+            CreateIcon(resDir + "/images/delete.png", 640.0f, 40.0f, 50.0f, 50.0f, CONTACT_DELETE_INDEX);
+            CreateLabel(name);
+        }
+    ```
+
+6. Implementation of `Visuals` used in the `ContactView` example is shown as follows. It is recommended to get yourself familiarized with [Visuals](visuals.md) before you start implementing your own `CustomView`.
```

Comment:
I would suggest something like this: 
What is your opinion?
```suggestion 
6. Implementation of `Visuals` used in the `ContactView` example is shown in the following example:

> [!IMPORTANT] 
> It is recommended to get yourself familiarized with [Visuals](visuals.md) before you start implementing your own `CustomView`.

Reply:
@annie-samsung When you do cross check @nawab-samsung's review, would you please leave your opinion about this also?
In my view, the "Important" mark is for the essential information required for user success. So to use to "It is recommended to get yourself familiarized with [Visuals](visuals.md) before you start implementing your own `CustomView`.", "Important" mark is too heavy. I think not editing it would be better.

**Reference :** https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md#alerts-note-tip-important-caution-warning

Reply:
@mijin85cho  I just wished to get an opinion regarding the same, 

My intention is/was as follows: 
The important element is for adding extra emphasis to general information. 

In this case, 
 A developer X must get familiarized with [Visuals](visuals.md) before implementing `CustomView`. 
So that implicitly becomes a requirement that is important to accomplish the task actually, so this is mandatory for user success. 

I would like to learn if my opinion is wrong.

Reply:
@nawab-samsung You understand is correct. In my opinion 3rd party developer should read Visuals page before start with custom view.

Reply:
@nawab-samsung, @mijin85cho @mihashco

Please find my opinion here:
https://github.com/Samsung/tizen-docs/pull/1344#discussion_r533127062